accounts-db/src/accounts_db.rs

```diff
@@ -455,7 +435,6 @@ pub struct AccountsDbConfig {
     pub account_indexes: Option<AccountSecondaryIndexes>,
     /// Base directory for various necessary files
     pub base_working_path: Option<PathBuf>,
```

Comment:
hash_calculation_pubkey_bins and num_hash_threads can be removed in future.

Reply:
Stumbled across this PR b/c it was linked for an unrelated thing, but I believe
https://github.com/anza-xyz/agave/blob/de646bd9a11ceb21bdfe255f8bb72b5799418f08/accounts-db/src/accounts_db.rs#L459-L460
can go away when this goes away
https://github.com/anza-xyz/agave/blob/de646bd9a11ceb21bdfe255f8bb72b5799418f08/accounts-db/src/accounts_db.rs#L1322-L1323
A quick scan shows we still have one caller:
https://github.com/anza-xyz/agave/blob/de646bd9a11ceb21bdfe255f8bb72b5799418f08/runtime/src/bank.rs#L4576

Not clear to me that we'd want to move to using the global pool, but I don't think we need to get into that too much further here. Rather, please give me a ping if/when y'all get around to adjusting this